packages/symfony-static-dumper/src/Command/DumpStaticSiteCommand.php

```diff
@@ -18,29 +19,44 @@ final class DumpStaticSiteCommand extends AbstractSymplifyCommand
 
     public function __construct(
         private SymfonyStaticDumperApplication $symfonyStaticDumperApplication,
-        ParameterBagInterface $parameterBag
+        private ParameterBagInterface $parameterBag
     ) {
         parent::__construct();
-
-        $projectDir = (string) $parameterBag->get('kernel.project_dir');
-
-        $this->publicDirectory = $projectDir . '/public';
-        $this->outputDirectory = getcwd() . '/output';
     }
 
     protected function configure(): void
     {
         $this->setName('dump-static-site');
         $this->setDescription('Dump website to static HTML and CSS in the output directory');
+
+        // Adding arguments options for the main command
+        $this->addOption('wdp-only', 'w', InputOption::VALUE_NONE, 'Will only process Controllers with Data Providers');
+        $this->addOption('public-dir', 'p', InputOption::VALUE_REQUIRED, 'Define the input public directory relative to the root project directory');
+        $this->addOption('output-dir', 'o', InputOption::VALUE_REQUIRED, 'Define the output directory relative to the execution of the comand');
```

Comment:
This looks like a good improvement. Could you extract it to own PR?

With default values from as 5th parameter:

```php
     $this->publicDirectory = $projectDir . '/public';
     $this->outputDirectory = getcwd() . '/output';
```

Reply:
yes, I'll do that! thanks